src/back/msl/writer.rs

```diff
@@ -34,6 +34,7 @@ const RAY_QUERY_FUN_MAP_INTERSECTION: &str = "_map_intersection_type";
 
 pub(crate) const MODF_FUNCTION: &str = "naga_modf";
 pub(crate) const FREXP_FUNCTION: &str = "naga_frexp";
+pub(crate) const NAGA_ISIGN_FUNCTION: &str = "naga_isign";
```

Comment:
nitpick: It would be more consistent with surrounding code to omit the `NAGA_*` prefix here:

```suggestion
pub(crate) const ISIGN_FUNCTION: &str = "naga_isign";
```

Reply:
Indeed, thanks!